plugins/inventory/foreman.py

```diff
@@ -68,6 +68,10 @@
       host_filters:
         description: This can be used to restrict the list of returned host
         type: string
+      per_page:
+        description: Determine the number of hosts returned by Foreman per page
```

Comment:
IMHO "per page" doesn't make sense in this context, as the user never sees any "pages". For them, this parameter is more of a "batch size". I'm fine with calling the parameter itself so, as that's also what the API receives, but I think the description text should explain what implications setting this would have.

Reply:
Yes, makes sense.. I will address that